src/amo/components/Addon/index.js

```diff
@@ -385,6 +386,22 @@ export class AddonBase extends React.Component {
     );
   }
 
+  renderContributeCard() {
+    const { addon } = this.props;
+
+    if (!addon || (addon && !addon.contributions_url)) {
+      return null;
+    }
+
+    return (
+      <Contribute
+        addonType={addon.type}
```

Comment:
Given that this card seems only to use props from `addon` I wonder if that should just be the only prop it takes. Then it can have a simpler proptype and can internally handle checking for the right attributes. It could also just return `null` if `addon` is undefined.